include/micm/process/process_set.hpp

```diff
@@ -21,7 +21,9 @@ namespace micm
     std::vector<std::size_t> number_of_products_;
     std::vector<std::size_t> product_ids_;
     std::vector<double> yields_;
-    std::vector<std::size_t> jacobian_flat_ids_;
+    //std::vector<std::size_t> jacobian_flat_ids_;
```

Comment:
remove this line

Reply:
I made std::vector<std::size_t> jacobian_flat_ids__ as protected member data, and removed the line.